src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const updatedRobot = {};
+
+  for (const key in robot) {
+    if (updatedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    updatedRobot[robot[key]] = key;
```

Comment:
```suggestion
    updatedRobot[robot[key]] = key;
```

[robot[key]] -- checklist p1

Reply:
@valerii-derkach what do you mean?
```suggestion
	const value = robot[key];
    updatedRobot[value] = key;
    
    is it right?
```